install/jaspic/other/testsuite.jtt

```diff
@@ -13,6 +13,6 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 #
```

Comment:
The copyright year needs to be updated from Copyright (c) 2018 to Copyright (c) 2018, 2020

Reply:
Done, thx!